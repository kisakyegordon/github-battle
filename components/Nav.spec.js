import React from 'react';
import { shallow, configure } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import Nav from './Nav';

configure({ adapter: new Adapter() });

describe("Test Results Component", () => {
    it("tests for the snapshot", () => {
        const wrapper = shallow(<Nav />);
        expect(wrapper).toMatchSnapshot();
    })
})