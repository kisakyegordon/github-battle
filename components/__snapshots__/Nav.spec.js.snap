// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Results Component tests for the snapshot 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Nav />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <li>
          <NavLink
            activeClassName="active"
            aria-current="page"
            exact={true}
            to="/"
          >
              Home  
          </NavLink>
        </li>,
        <li>
          <NavLink
            activeClassName="active"
            aria-current="page"
            to="/battle"
          >
              Battle  
          </NavLink>
        </li>,
        <li>
          <NavLink
            activeClassName="active"
            aria-current="page"
            to="/popular"
          >
              Popular  
          </NavLink>
        </li>,
      ],
      "className": "nav",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <NavLink
            activeClassName="active"
            aria-current="page"
            exact={true}
            to="/"
          >
              Home  
          </NavLink>,
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "activeClassName": "active",
            "aria-current": "page",
            "children": "  Home  ",
            "exact": true,
            "to": "/",
          },
          "ref": null,
          "rendered": "  Home  ",
          "type": [Function],
        },
        "type": "li",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <NavLink
            activeClassName="active"
            aria-current="page"
            to="/battle"
          >
              Battle  
          </NavLink>,
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "activeClassName": "active",
            "aria-current": "page",
            "children": "  Battle  ",
            "to": "/battle",
          },
          "ref": null,
          "rendered": "  Battle  ",
          "type": [Function],
        },
        "type": "li",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <NavLink
            activeClassName="active"
            aria-current="page"
            to="/popular"
          >
              Popular  
          </NavLink>,
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "activeClassName": "active",
            "aria-current": "page",
            "children": "  Popular  ",
            "to": "/popular",
          },
          "ref": null,
          "rendered": "  Popular  ",
          "type": [Function],
        },
        "type": "li",
      },
    ],
    "type": "ul",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <li>
            <NavLink
              activeClassName="active"
              aria-current="page"
              exact={true}
              to="/"
            >
                Home  
            </NavLink>
          </li>,
          <li>
            <NavLink
              activeClassName="active"
              aria-current="page"
              to="/battle"
            >
                Battle  
            </NavLink>
          </li>,
          <li>
            <NavLink
              activeClassName="active"
              aria-current="page"
              to="/popular"
            >
                Popular  
            </NavLink>
          </li>,
        ],
        "className": "nav",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <NavLink
              activeClassName="active"
              aria-current="page"
              exact={true}
              to="/"
            >
                Home  
            </NavLink>,
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "function",
            "props": Object {
              "activeClassName": "active",
              "aria-current": "page",
              "children": "  Home  ",
              "exact": true,
              "to": "/",
            },
            "ref": null,
            "rendered": "  Home  ",
            "type": [Function],
          },
          "type": "li",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <NavLink
              activeClassName="active"
              aria-current="page"
              to="/battle"
            >
                Battle  
            </NavLink>,
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "function",
            "props": Object {
              "activeClassName": "active",
              "aria-current": "page",
              "children": "  Battle  ",
              "to": "/battle",
            },
            "ref": null,
            "rendered": "  Battle  ",
            "type": [Function],
          },
          "type": "li",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <NavLink
              activeClassName="active"
              aria-current="page"
              to="/popular"
            >
                Popular  
            </NavLink>,
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "function",
            "props": Object {
              "activeClassName": "active",
              "aria-current": "page",
              "children": "  Popular  ",
              "to": "/popular",
            },
            "ref": null,
            "rendered": "  Popular  ",
            "type": [Function],
          },
          "type": "li",
        },
      ],
      "type": "ul",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
